pipelines/bootstrap.sh

```diff
@@ -163,16 +163,34 @@ function clean_openshift_pipelines() {
 
 }
 
-export BASEDIR=$(dirname "$0")
+
+if [[ "${0}" == *"bootstrap.sh" ]]; then
+    echo "Running locally"
+    export WORKDIR=$PWD
+
+    if [[ $PWD == *'pipelines' ]]; then
```

Comment:
The suggested edit above should give you the path to the bootstrap.sh, from there you can use relative paths to build the `PIPELINES_DIR`